### ISSUE TITLE
Define Accessibility Requirements per Project Phase / Step

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9087]
**Description:** 

**Notes:** 

**AC**

**Comment by brea11y** [comment]
@bischoffa For this ticket and the one about accessibility artifacts, where should I make these updates / recommendations? Would the Google slides presentation that you previously shared work or should we begin the process of moving that information to the doc site and I can add it there?

**Comment by bischoffa** [comment]
@brea11y lets create a separate doc for now and you can do that in Google with whatever program you would like. I would just recommend to reference what steps the work goes to.

**Comment by brea11y** [comment]
You got it! I'll get that together today!

**Comment by brea11y** [comment]
@bischoffa – I have created a [Google doc here](https://docs.google.com/document/d/1VaZuurcgR5Bk3OpIWGT8bRyWoCzgy-P0grqAtDJQbsA/edit?usp=sharing) where I have imported the steps from the Google slides presentation. In each step, I have highlighted areas in yellow where accessibility should be included and aqua/blue where an artifact will be required. I will also drop a link to this document in the ticket that is for artifacts (but am not tagging you in that one to avoid spamming you).

**Comment by brea11y** [comment]
Will review this with Therese during 1:1 (along with the artifacts in the other tickets). Will create new tickets based on that conversation.
